lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+
+import {
+  Checkbox,
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+import { TextField } from '../Fields';
+import { TooltippedControl } from '../TooltippedControl';
+import { validateRequired } from '../utils';
+import { FieldCurrency } from '../Currency';
+import CurrentExchangeRate from './CurrentExchangeRate';
+
+const CurrencyExchangeRateFields = ({
+  change,
```

Comment:
change fn is from RFF? or it's a callback?

Reply:
is fn from RFF. Could be used from `useForm`

Reply:
in case this just RFF fn, it could be get right here from the hook.